person.rb

```diff
@@ -15,9 +21,22 @@ def can_use_services?
     true
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
+
+person = Person.new(22, name: 'maximilianus')
+puts person.correct_name
+
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name
+
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+puts capitalized_trimmed_person.correct_name
```

Comment:
After fixing the errors and testing your code, here are the outputs:

![image](https://github.com/danielochuba/oop-school-library/assets/77069011/3a142e11-3371-46bf-89d2-e78891ef7c80)

- [ ] Please take in mind that the `CapitalizeDecorator` should "capitalize" the name, so only the first letter should be a capital one.


```rb
# Expected Results:

person = Person.new(22, name: 'maximilianus')
puts person.correct_name # ---> maximilianus

capitalized_person = CapitalizeDecorator.new(person)
puts capitalized_person.correct_name # ---> Maximilianus

capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
puts capitalized_trimmed_person.correct_name # ---> Maximilian

```

Please consider fixing this! 🤓 🔧

Reply:
@anagudelogu thanks alot